### ISSUE TITLE
XMLWriter.close() does not close the underlying writer

### ISSUE THREAD

**Issue by GoogleCodeExporter** [unlockha/beanio#63]
```
As discussed in the forum topic: 
https://groups.google.com/forum/?fromgroups=#!topic/beanio/_q3DKkphxJk

```

Original issue reported on code.google.com by `kevin.s...@gmail.com` on 23 Feb 2013 at 3:28

**Comment by GoogleCodeExporter** [comment]
```
Fixed for 2.0.4.
```

Original comment by `kevin.s...@gmail.com` on 23 Feb 2013 at 3:29

**Comment by GoogleCodeExporter** [comment]
Original comment by `kevin.s...@gmail.com` on 6 Mar 2013 at 3:14
- Changed state: **Fixed**
